### ISSUE TITLE
Azuracast "slow or hanging"? Both stable and rolling release. 

### ISSUE THREAD

**Issue by djwoefgithub** [AzuraCast/AzuraCast#5149]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.15.2

### What happened?

First of all, let me express that I really enjoy this nice piece of software and that I appreciate all the work you guys put in!

Now, today I updated both instances (one on stable, the other one on rolling release) to the latest versions.

Maybe its only me, in case no one else recognizes this, then please delete this bug! Sorry!  Then its something weird in my instances or something.

But I see strange behaviour on both instances:
-The stable 0.15.2  plays only one song over and over again no matter how I adapt the playlist. I just can't skip to the next song.
The GUI is slow and I see the mysql cpu usage going op to 60-70 %.

-The rolling release (released today) the gui is not responsive at all. After giving username/passwd nothing happens. No stream.. Everything times out. On the  instance itself(cli) the cpu usage of mysql goes upto 90%. 

Does anybody recognize this behaviour ?

### Relevant log output

```shell
Please let me know if you want any logging. I am happy to help.
```

**Comment by SlvrEagle23** [comment]
@djwoefgithub In the _very_ latest rolling release (i.e. from yesterday), I was able to identify one potential issue with MariaDB causing it to lock up in some circumstances, but it certainly shouldn't be using that kind of CPU load in regular usage. I think this may be the culprit behind a lot of people's reports of CPU overrun, but the problem is I don't currently have enough diagnostic information to pin down the cause.

One thing I like to do on my instances just to see what the problems are is to run:

```
cd /var/azuracast
./docker.sh db
```

to launch into the database console, then run:

```
SHOW PROCESSLIST;
```

from time to time, especially when CPU is high, to see what queries are running, how long they've taken, etc. That can help to identify bottlenecks in performance.

**Comment by iplayradionetworks** [comment]
On our server i also see a CPU incresae too.... in comparison with other software like Centovacast for example. Centovacast was only consuming 20-30% max.... all day long with same amount of radios. And on Azuracast i see 35-50% max... this is not right i believe. Unless its meant to operate like this.

**Comment by nrq** [comment]
Good day, I noticed the GUI is quite slow today.  I used the auto script on Linode on 1GB vps to install Azuracast.  After a two months, it stopped casting.   I got an error with Maria db in the docker:
azuracast_web  |   SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for mariadb fa
azuracast_web  |   iled: Name or service not known

So I rebooted the vps,  same problem.  So, I then increased the ram to 2GB.  It started working, and the gui is still slow and ram is 100% used.  I have about 4 listeners.  
https://radio.nurulquran.com/public/nurulquran/ondemand  is slow and gives error on the search.

Do I need to use a better server or is it something else?

Thanks.
